src/actions/recommenderActions.js

```diff
@@ -0,0 +1,9 @@
+export const likeEvent = (userId, categories) => dispatch => {
+    const requestOptions  = {
+        method: 'POST',
+        headers: {'Content_Type': 'application/json'},
+        body: JSON.stringify({userId, categories})
+    }
+    return fetch('/rateEvent', requestOptions)
```

Comment:
@lamchiminhmark Does returning the fetch call here actually ever dispatch an action to Redux? Or are you not attempting to dispatch an action?